wat

```diff
@@ -22,6 +26,13 @@ mark_lines() {
 
 : ${PACMAN_LOG:=/var/log/pacman.log}
 
+while [ $# -gt 0 ]; do
+    case $1 in
+        -i|--installed) SHOW_INSTALLED=1; shift ;;
```

Comment:
An all capital variable usually indicates and environment variable, which this is not. Please lowercase it.